ztp/ran-crd/site-config-crd.yaml

```diff
@@ -318,6 +318,14 @@ spec:
                                 - UEFI
                                 - UEFISecureBoot
                                 - legacy
+                            automatedCleaningMode:
+                              description: When set to disabled, automated cleaning will be avoided
+                                during provisioning and deprovisioning.
+                              type: string
+                              default: disabled
+                              enum:
+                                - metadata
```

Comment:
what metadata ? I think this should be enable

Reply:
Hmm - I thought we are supposed to use the same definitions as in the original CRD, in this case BareMetaHost. This is how the BMH defines this field:
- https://github.com/metal3-io/baremetal-operator/blob/main/apis/metal3.io/v1alpha1/baremetalhost_types.go#L418
- https://github.com/metal3-io/baremetal-operator/blob/main/config/crd/bases/metal3.io_baremetalhosts.yaml#L77

Do we want to have the enum contain "enabled" and "disabled" instead of the original BMH definition? @imiller0 & @serngawy, kindly advise.

Reply:
Confirmed with baremetal folks that with the metadata option the partitioning table is removed, but the disk isn't fully wiped. I was wrong above and this is NOT related to this [field](https://github.com/metal3-io/baremetal-operator/blob/main/config/crd/bases/metal3.io_baremetalhosts.yaml#L245).

I updated the testcases accordingly. Thanks, @serngawy